src/SkeletalTrapezoidationEdge.h

```diff
@@ -1,9 +1,14 @@
-//Copyright (c) 2019 Ultimaker B.V.
+//Copyright (c) 2020 Ultimaker B.V.
 
 
 #ifndef VORONOI_QUADRILATERALIZATION_EDGE_H
 #define VORONOI_QUADRILATERALIZATION_EDGE_H
```

Comment:
Hmm, the header guard has a different name than the file name.

Reply:
I should be less laser focused and do some more scouting during tickets sometimes... :-)